commit 39dc7cdc013d391b79ed7b130dc940f8576aaf89 (refs/changes/74/1674/2)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-03-16T21:18:33+00:00 (2 years, 7 months ago)
    
    DOCKER-1019 cli-local-volumes.test.js fails in nightly
    Reviewed by: Trent Mick <trent.mick@joyent.com>
    Approved by: Trent Mick <trent.mick@joyent.com>

diff --git a/test/integration/cli-local-volumes.test.js b/test/integration/cli-local-volumes.test.js
index c7f6670..7c43f22 100644
--- a/test/integration/cli-local-volumes.test.js
+++ b/test/integration/cli-local-volumes.test.js
@@ -51,9 +51,9 @@ test('setup', function (tt) {
 
 test('docker local volumes', function (tt) {
     var containerWithLocalVolName =
-        common.makeResourceName('local-volume-test-container-with-local-vol');
+        common.makeContainerName('local-volume-test-container-with-local-vol');
     var mountingContainerName =
-        common.makeResourceName('local-volume-test-mounting-container');
+        common.makeContainerName('local-volume-test-mounting-container');
 
     tt.test('creating container with local volume should succeed',
         function (t) {
